JRuby.md

```diff
@@ -0,0 +1,117 @@
+## JRuby
+
+> The Ruby Programming Language on the JVM
+
+This document serves as a getting-started guide for those wishing to use
+[Sorbet](https://sorbet.org) with [JRuby](https://www.jruby.org/).
+
+The use of Sorbet on the JVM is not **officially** supported by the Sorbet
+development team however this document services as a getting started guide
+for those who wish to trot on a unsupported path.
+
+### Performance
+
+TBD
+
+### Gotchas
+
+#### RBI for Java Code
+
+The `srb` tool can auto-generate RBI files for your gems however it cannot
+do so for your Java code :(
```

Comment:
Non-blocking request:

- Can we make the tone more neutral? i.e., no `:(`, no first person.

Reply:
Will comb over